lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -100,13 +104,26 @@ RESET_IDENTITY:
     lambda: reset-identity
   events:
     next:
-      targetState: IPV_IDENTITY_START_PAGE
+      targetState: PYI_DETAILS_DELETED_PAGE
```

Comment:
This will affect existing journeys that go through `RESET_IDENTITY` so I think we'll need a new state specifically for the user initiated reset flow

Reply:
Do we not want the delete confirmation screen to show everywhere we delete a user's details?

Reply:
Oh I see, you are talking about times when we delete the users details without the user initiating it themselves?